src/Controller/ResultController.php

```diff
@@ -75,13 +69,14 @@ public function showResults(Request $request, array $requestAttributes)
      * @param array $requestAttributes
      * @return Response
      */
```

Comment:
Function name does not match logic.
Logic tells me you load the results for a single quiz.
The name tells me you show the results for multiple quizzes.
Please fix this. 
Please also add space before and after operators.  `'key'=>$var`   will look like this `'key' => $var`